milestone-3/frontend/src/components/home.tsx

```diff
@@ -0,0 +1,30 @@
+import "./home.css";
+import RecipePreview from "./recipePreview";
+import recipes from "../recipeData.json";
+import {useState, useEffect} from 'react';
+import {Recipe} from '../types';
+
+export default function Home() {
+  const [externalRecipes, setExternalRecipes] = useState<Recipe[]>([]);
+  useEffect(() => {
+    fetch("https://bootcamp-milestone-4.onrender.com/recipe")
+      .then((res) => res.json())
+      .then((data) => setExternalRecipes(data));
+  }, []);
```

Comment:
add a .catch() to the end of your .then chains so you can catch and handle any errors you receive with your api call.